### ISSUE TITLE
2.7.6 patch release plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#2892]
- #2741
- #2714 
- #2775
- #2722 
- #2719
- #2756 
- #2839
- #2863 
- #2879 
- #2885 
- #2882 
- #2891

## Pick list
```
git cherry-pick db3a59be17ce091715b9c948d643c175a62a5a79
git cherry-pick 802248d6aa633c9aada2c041f37613ef2a1fa5ac
git cherry-pick 4f23fcdd4419a42f4f1972bc3686621ee64dad71
git cherry-pick da389a7a91ae0f0cd50c19d387de3195edf8c606
git cherry-pick 24421b0da2a21f310a6bf2899640bcdce5684dbf
git cherry-pick 0c27e516f300556a29fa11d47721066be87b2045
git cherry-pick 35e924e575a36794f35d43deb534cafb255c82dd
git cherry-pick 984049f290389b69948aac64bc71e5169dbcd015
git cherry-pick 4f6c6ea9423fb4b71e6d841d09def50a0224abe6
git cherry-pick 68f354b627af19f23d27578e8b8b2cbded9f3da8
git cherry-pick 02463aea7e2ee96c977a1def65ac12e29739b453
git cherry-pick 25931f6f3385bd98b88b56952e682d00f9385548
```

**Comment by joeparsons** [comment]
## Release notes snippet
`git log 2.7.5..HEAD --pretty='format:- %s'`
```
_Note: This release contains a security update for the Paragraphs admin module addressing a moderately critical security vulnerability ([SA-CONTRIB-2023-049](https://www.drupal.org/SA-CONTRIB-2023-049))._

- Closes #2890: Update paragraphs_admin to 8.x-1.5 for SA-CONTRIB-2023-049. (#2891)
- Closes #2732 Alternating Split Screen paragraphs don't line up in the center (#2882)
- Closes #2884: Address ProboCI drush issues (#2885)
- Closes #2878 Bootstrap Barrio Subtheme not D10 Compatible. (#2879)
- Closes #2786: Update phonebook/directory links in global footer. (#2863)
- Closes #2838 Fix failing create-release workflow. (#2839)
- Closes #2755 Update create release workflow ahead of API changes. (#2756)
- Closes #2668: Add click to copy functionality to Marketing Cloud integration (#2722)
- Closes #2718: Update pagination sr-only heading levels (#2719)
- Closes #2774 Undefined array key warning with Text on Media paragraphs (#2775)
- Closes #2710 Remove Arizona Bootstrap overrides. (#2714)
- Update drupal/cas requirement from 2.1.0 to 2.2.0 (#2741)
- Prepare 2.7.x branch for new minor release branch 2.8.x (#2813)
- Back to dev.

```

**Comment by joeparsons** [comment]
Release created: https://github.com/az-digital/az_quickstart/releases/tag/2.7.6
